app.js

```diff
@@ -0,0 +1,29 @@
+//setup express server
+const express = require("express");
+const app = express();
+
+//middleware
+app.use(express.json());
+
+//routes
+const { getApi } = require("./controllers/api.controller");
+
+app.get("/api", getApi);
+
+// 404 handler
+app.all("/*splat", (req, res) => {
+  res.status(404).send({ msg: "404 Not Found" });
+});
+
+// erroe handlers
+app.use((err, req, res, next) => {
+  if (err.code === "22P02") {
+    return res.status(400).send({ msg: "Bad Request" });
+  } else if (err.status) {
+    return res.status(err.status).send({ msg: err.msg });
+  } else {
+    return res.status(500).send({ msg: "500 Internal Server Error" });
+  }
```

Comment:
We don't need any of this code yet to pass the test, remember to only push code necessary to the task 👍